setup.py

```diff
@@ -31,11 +31,11 @@
             'html = aleph.ingest.html:HtmlIngestor',
             'djvu = aleph.ingest.djvu:DjVuIngestor',
             'img = aleph.ingest.image:ImageIngestor',
-            'email = aleph.ingest.email:EmailFileIngestor',
-            'mbox = aleph.ingest.email:MboxFileIngestor',
+            'email = aleph.ingest.email_:EmailFileIngestor',
```

Comment:
I don't know what in aleph used requests before or whether it worked - I guess no ingestor was using `requests`, but now something in aleph.ingest is (the code I added) which means the email module is picked up incorrectly.